spec/cyanobyte.schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://github.com/cyanobyte/cyanobyte.schema.json",
+  "title": "CyanoByte Specification Validator",
+  "description": "This file is used to validate whether a given document meets the CyanoByte specification.",
+  "type": "object",
+  "properties": {
+    "cyanobyte": {
+      "description": "The version of the CyanoByte specification this file follows.",
+      "type": "string"
+    },
+    "info": {
+      "description": "Info block of the file",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "The title of the document. This should be used as the class name in the auto-generated code and therefore should be short (e.g. the part name).",
+          "type": "string"
+        },
+        "description": {
+          "description": "A description of the document.",
+          "type": "string"
+        },
+        "contact": {
+          "description": "Contact info for the maintainer.",
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "Name of the maintainer.",
+              "type": "string"
+            },
+            "url": {
+              "description": "URL of the maintainer organization.",
+              "type": "string",
+              "format": "uri"
```

Comment:
Your test input `example.com` does not to work here. I recommend `http://example.com`.

> String 'example.com' does not validate against format 'uri'.

Reply:
Ahh woops, I made that change, but uploaded a slightly older version.